apps/settings/test/unit/panels/root/stk_item_test.js

```diff
@@ -86,10 +87,6 @@ suite('STK (Main menu) >', function() {
           }})});
     });
 
-    teardown(function() {
-      mocksHelper.teardown();
-    });
-
     test('One entry into the STK applications list', function() {
       assert.equal(document.getElementById('icc-entries').childElementCount, 1);
     });
```

Comment:
I think there are something that we can refactor here to make code not so messy, but it's kinda hard to explain, so let me provide a diff here.